<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemSearchClass</class>
 <widget class="QDialog" name="ItemSearchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemSearch</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabAttributeTypes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="attributes">
      <attribute name="title">
       <string>By Attributes</string>
      </attribute>
      <layout class="QGridLayout" name="attributeGrid">
       <property name="margin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="1" column="0">
        <widget class="ItemAttributes" name="itemAttributes" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboVersions"/>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkAdvanced">
         <property name="text">
          <string>Advanced</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>By Value</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="buttonID">
         <property name="text">
          <string>By ID</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="buttonProperty">
         <property name="text">
          <string>By Property</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Properties</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>5</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkWidth">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="valueWidth">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkHeight">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="valueHeight">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkCropsize">
            <property name="text">
             <string>Cropsize</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="valueCropsize">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkLayers">
            <property name="text">
             <string>Layers</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="valueLayers">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkXDiv">
            <property name="text">
             <string>X Repeat</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="valueXDiv">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkYDiv">
            <property name="text">
             <string>Y Repeat</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="valueYDiv">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="checkZDiv">
            <property name="text">
             <string>Z Repeat</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="valueZDiv">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="checkAnimations">
            <property name="text">
             <string>Animations</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="valueAnimations">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="valueID">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>2147483647.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="color">
      <attribute name="title">
       <string>By Color</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioColorRed">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioColorGreen">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioColorBlue">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioColorYellow">
         <property name="text">
          <string>Yellow</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="radioColorMagenta">
         <property name="text">
          <string>Magenta</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="radioColorCyan">
         <property name="text">
          <string>Cyan</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Intensity</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="spinIntensity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3" colspan="2">
    <widget class="QTreeView" name="itemView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="indentation">
      <number>10</number>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="checkFilters">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Exclude</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="filterItems">
        <property name="text">
         <string>Items</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="filterOutfits">
        <property name="text">
         <string>Outfits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="filterEffects">
        <property name="text">
         <string>Effects</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="filterProjectiles">
        <property name="text">
         <string>Projectiles</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QGroupBox" name="checkCombinations">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Combine</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="combineAttributes">
        <property name="text">
         <string>Attributes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="combineValues">
        <property name="text">
         <string>Values</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="combineColors">
        <property name="text">
         <string>Colors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonSelect">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="buttonFind">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ItemAttributes</class>
   <extends>QWidget</extends>
   <header>itemattributes.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkWidth</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkHeight</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueHeight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCropsize</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueCropsize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkLayers</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueLayers</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkXDiv</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueXDiv</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkYDiv</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueYDiv</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkZDiv</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueZDiv</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkAnimations</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueAnimations</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonProperty</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonID</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFilters</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterItems</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFilters</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterOutfits</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFilters</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterEffects</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFilters</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterProjectiles</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>462</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCombinations</sender>
   <signal>toggled(bool)</signal>
   <receiver>combineAttributes</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>583</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCombinations</sender>
   <signal>toggled(bool)</signal>
   <receiver>combineValues</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>604</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCombinations</sender>
   <signal>toggled(bool)</signal>
   <receiver>combineColors</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>630</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
